sdcm/results_analyze/__init__.py

```diff
@@ -448,7 +447,7 @@ def check_regression(self, test_id, data, is_gce=False, node_benchmarks=None):
         dataset_size = re.search(r'(\d{3}gb)', config_files).group() or 'unknown size'
 
         subject = (f'{self._get_email_tags(doc, is_gce)} Performance Regression Compare Results '
-                   f'({self.test_name_for_email_subject} {dataset_size}) -'
+                   f'({email_subject_postfix} {dataset_size}) -'
```

Comment:
with this change, `latency during upgrade` is now broken

Reply:
move reasons to move it away

Reply:
this line is now removed, and moved to configuration.